Author: George Kiagiadakis, Diane Trout
Description: Force a SO version number on libktpchat.
 later update so the SO version number defaults to the package version.

--- a/lib/CMakeLists.txt
+++ b/lib/CMakeLists.txt
@@ -52,6 +52,8 @@
 
 kde4_add_library(ktpchat SHARED ${ktpchat_SRCS})
 
+set_target_properties(ktpchat PROPERTIES SOVERSION 0 VERSION ${KTP_TEXT_UI_VERSION})
+
 target_link_libraries(ktpchat
     ${KDE4_KDECORE_LIBS}
     ${KDE4_KIO_LIBS}
